case-lib/lib.sh

```diff
@@ -252,6 +252,16 @@ func_lib_get_tplg_path()
     return 0
 }
 
+func_lib_check_pa()
+{
+    if ! pactl stat &>/dev/null; then
+        dloge "pulseaudio doesn't start correctly"
+        return 1
+    fi
+
+    return 0
```

Comment:
In the test cases, why do we care the interactive shell?

Reply:
In my new patch, I will use :
pactl stat &>dev/null || {
dloge ...
return 1
}

I still use "&>dev/null" because error message `pactl stat` doesn't show explicit information that `pactl stat` fails. It doesn't help for developer to identity which fails quickly. Instead, I will use `dloge` to export the useful information for sof-test developer.